### PR TITLE
feat: add time datetime for valley

### PR DESCRIPTION
**Pull request by Lash-L**
Adds Unknown Method error

Time for the dnd and valley.

I need to create a setter for those, but I need to do some reverse engineering as I can't get the set command to actually change the timer.

### CODE REVIEW

File: roborock/api.py [hunk 1]
```diff
@@ -189,6 +191,8 @@ async def _async_response(self, request_id: int, protocol_id: int = 0) -> tuple[
             queue = RoborockFuture(protocol_id)
             self._waiting_queue[request_id] = queue
             (response, err) = await queue.async_get(QUEUE_TIMEOUT)
+            if response == "unknown_method":
```
Comment (humbertogontijo) [inline roborock/api.py:194]: I probably got this a feel times but I cant remember if the response is just this code unknown_method. Is it?

Comment (Lash-L) [reply roborock/api.py:194]: At least sometimes it is. That's the response I got for valley timer as example

Comment (Lash-L) [reply roborock/api.py:194]: I'm not sure why the response is that instead of the err

Comment (Lash-L) [reply roborock/api.py:194]: I would assume it always returns in English and never changes but I'm not 100% on that

Comment (humbertogontijo) [reply roborock/api.py:194]: Valley timer does work for you?

Comment (Lash-L) [reply roborock/api.py:194]: It does not. It's not an option for my device

**Review by humbertogontijo — APPROVED** [review]

**Pull request merged**